corehq/apps/reports/standard/tableau.py

```diff
@@ -81,9 +81,9 @@ def get(self, request, *args, **kwargs):
 def get_tableau_server_ticket(request, domain):
     from requests_toolbelt.adapters import host_header_ssl
     visualization_data = {data: value[0] for data, value in dict(request.POST).items()}
-    server_name = visualization_data.pop('server_name')
-    validate_hostname = visualization_data.pop('validate_hostname')
-    target_site = visualization_data.pop('target_site')
+    server = TableauServer.objects.get(domain=domain)
```

Comment:
nit: can we use this below for `if TableauServer.objects.get(domain=domain).get_reports_using_role:`